source/language.c

```diff
@@ -795,4 +795,104 @@ const char* const g_strings[StrId_Max][16] =
 			"已完成 %zu / %zu KiB"
 		),
 	},
+
+	[StrId_NetSender] =
+	{
+		STR_EN("3dslink NetSender"),
+		STR_ES(""),
+		STR_DE(""),
+		STR_FR("Envoyeur de programme 3dslink"),
+		STR_IT(""),
+		STR_JP(""),
+		STR_PT(""),
+		STR_NL(""),
+		STR_KO(""),
+		STR_RU(""),
+		STR_ZH(""),
+		STR_TW(""),
+	},
+
+	[StrId_NetSenderUnavailable] =
+	{
+		STR_EN("The NetSender is currently unavailable."),
+		STR_ES(""),
+		STR_DE(""),
+		STR_FR("L'envoyeur de programme 3dslink est indisponible."),
+		STR_IT(""),
+		STR_JP(""),
+		STR_PT(""),
+		STR_NL(""),
+		STR_KO(""),
+		STR_RU(""),
+		STR_ZH(""),
+		STR_TW(""),
+	},
+
+	[StrId_NetSenderInvalidIp] =
+	{
+		STR_EN("Invalid IP address."),
+		STR_ES(""),
+		STR_DE(""),
+		STR_FR("Address IP invalide."),
+		STR_IT(""),
+		STR_JP(""),
+		STR_PT(""),
+		STR_NL(""),
+		STR_KO(""),
+		STR_RU(""),
+		STR_ZH(""),
+		STR_TW(""),
+	},
+	
+	[StrId_NetSenderActive] =
```

Comment:
This is a duplicate of StrId_NetLoaderActive.

Reply:
not exactly a duplicate because the format part for the ip and port was removed. I believe they should stay separate, as showing the ip is useless for the sender